static/js/src/publisher-admin/routes/root.tsx

```diff
@@ -19,7 +20,25 @@ function Root() {
     }
   }, [packageData]);
 
-  return (
+  const isCollaborationTab = location.pathname.endsWith("/collaboration");
```

Comment:
That's very specific and quite fragile workaround.

A bit safer approach would be to move the application layout divs to the actual view of the collaboration tab which I guess is this one [Collaboration.tsx](https://github.com/canonical/charmhub.io/blob/main/static/js/src/publisher-admin/pages/Collaboration/Collaboration.tsx#L91) and wrapping contents there in the application layout stuff.
This will not be exactly the same as before (it will be a bit more nested), but from a quick check it seems to work and isolates the application layout a bit better (to the view that actually uses it).